pyproject.toml

```diff
@@ -37,7 +37,7 @@ types-pytz = ">= 2022.1.1"
 mypy = "0.990"
 pyarrow = ">=9.0.0"
 pytest = ">=7.1.2"
-pyright = ">=1.1.278"
+pyright = "<=1.1.279"
```

Comment:
Maybe add a pyright issue number, if it exists

Reply:
Latest main uses `pyright` >= 1.1.281, so this is resolved.